anvilPoll2024MainAnalysis.Rmd

```diff
@@ -602,6 +657,17 @@ Finally, we'll reverse the x-axis so that most important is on the right and lea
 </details>
 
 ```{r, message=FALSE, echo = FALSE}
+
```

Comment:
I can see you were playing around with the axes here, but we end up with the same (or very similar?) plot in the report. I'd just choose one so extra code isn't distracting.